index.js

```diff
@@ -110,8 +110,8 @@ module.exports = (app, options) => {
         })
 
         const contentType = (res.headers['content-type'] || res.headers['Content-Type'] || '').split(';')[0]
-        const contentEncoding = res.headers['content-encoding'] || res.headers['Content-Encoding'] || ''
-        const isBase64Encoded = options.binaryMimeTypes.indexOf(contentType) > -1 || !!contentEncoding
+        const customBinaryCheck = typeof options.isBinary === 'function' && !!options.isBinary(res)
```

Comment:
Do not correct the return here. Let it as is.
```suggestion
        const customBinaryCheck = typeof options.isBinary === 'function' && options.isBinary(res)
```